packages/teams-js/src/private/interfaces.ts

```diff
@@ -132,6 +132,15 @@ export interface FilePreviewParameters {
    */
   type: string;
 
+  /**
+   * @hidden
+   * The size of the file in bytes.
+   *
+   * @internal
+   * Limited to Microsoft-internal use
+   */
+  size?: string;
```

Comment:
>string [](http://example.com/codeflow?start=9&length=6)

Wouldn't a count of bytes be more faithfully represented in a `number` type?

Reply:
With string we can handle any value without worrying for limit and null values can be checked.

Reply:
Can you say more about which limits are concerning? The maximum value that `number` can hold is 9,007,199,254,740,991, and since that is going to be in bytes in this context, it means that your interface would be limited to files that are no larger than roughly 9000 terabytes. I understand the value of future-proofing, but it doesn't seem like that developers will be handling files that large in their Teams apps anytime soon? If you were concerned with being able to handle larger files, you could instead use [BigInt](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/BigInt) which has no limit and still faithfully represents that this a number and not a string.

Can you elaborate on the scenarios for null-value checking that work with `string` but don't work with `number`? If this property was defined as `sizeInBytes?: number` that would allow all the same checks that I expect would commonly be used. Are you specifically concerned with checking for `null` or did you mean `undefined`?

Reply:
I was specifically concerned for null and also where this property will get consumed is specified at string, so I used the same type here. But you are right makes more sense as a number, the consumer can convert as per there need. I will modify this to a number.